views/signup.handlebars

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
```

Comment:
This whole document should be set be loaded into the {{{ body }}} of the main template.